DarkMagician/DarkMagician.Console/Program.cs

```diff
@@ -0,0 +1,79 @@
+﻿Console.ForegroundColor = ConsoleColor.Cyan;
+Console.WriteLine("WELCOM TO DARK WORLD! ");
+Console.WriteLine("PLACE FULL OF MONSTERS, MAGIC AND... HOT ANIMATED GIRLS");
+Console.WriteLine();
+Console.WriteLine("WHAT IS YOUR NAME?");
+Console.ForegroundColor = ConsoleColor.White;
+var name = Console.ReadLine();
+Console.ForegroundColor = ConsoleColor.Cyan;
+Console.WriteLine("SO..." + name + " WHATS YOUR CLASS? WARRIOR? ROGUE? OR MAGICIAN?");
+Console.ForegroundColor = ConsoleColor.White;
+var playerClass = Console.ReadLine();
+if (playerClass == "WARRIOR")
+{
+    Console.ForegroundColor = ConsoleColor.Red;
+
+    Console.WriteLine("ERROR UNABLE TO CHOOSE THIS CLASS");
+    Console.WriteLine();
+
+    Console.ForegroundColor = ConsoleColor.Cyan;
+    Console.WriteLine("SOMETHINGS WRONG... CHOOSE AGAIN");
+    Console.ForegroundColor= ConsoleColor.White;
+    playerClass = Console.ReadLine();
+}
+Console.ForegroundColor = ConsoleColor.White;
+if (playerClass == "ROGUE")
+{
+    Console.ForegroundColor = ConsoleColor.Red;
```

Comment:
możesz użyć funkcji do wyświetlania tekstu jako X, w której masz wyświetlanie tego tekstu i kolorowanie go

np `PrintAsGod()`, `PrintAsDupa()` itd

będziesz tego używać jako przykładowo `PrintAsError("ERROR UNABLE TO CHOOSE THIS CLASS")` zamiast 
```csharp
Console.ForegroundColor = ConsoleColor.Red;
Console.WriteLine("ERROR UNABLE TO CHOOSE THIS CLASS");
```

zaoszczędzi ci to sporo miejsca w kodzie i kod będzie wyglądał dużo czytelniej, np 

```csharp
PrintAsError("ERROR UNABLE TO CHOOSE THIS CLASS");
PrintAsGod("SOMETHINGS WRONG... CHOOSE AGAIN");
playerClass = GetInputFromUser();
```

w 3 linijkach masz to samo, co wcześniej miałeś w 6, plus spacje czyli jeszcze więcej 

przykładowa funkcja to może być
```csharp
void PrintAsGod(string text)
{
    Console.ForegroundColor = ConsoleColor.Cyan;
    Console.WriteLine(text);
}
```

dzięki temu nigdy nie zapomnisz o prawidłowym ustawieniu koloru tekstu, jeśli wybierzesz wypisanie tekstu za pomocą `PrintAsGod()` to tekst ZAWSZE będzie cyjanowy, bo `PrintAsGod` wewnątrz siebie ustawia kolor tekstu na cyjanowy

Reply:
```csharp
string GetUserInput()
{
    Console.ForegroundColor = ConsoleColor.Cyan;
    var result = Console.ReadLine();
    return result;
// alternatywnie
// return Console.ReadLine();
}
```